curations/maven/mavencentral/antlr/antlr.yaml

```diff
@@ -0,0 +1,16 @@
+coordinates:
+  name: antlr
+  namespace: antlr
+  provider: mavencentral
+  type: maven
+revisions:
+  2.7.7:
+    described:
+      sourceLocation:
+        name: antlr
+        namespace: antlr
+        provider: mavencentral
+        revision: 2.7.7
```

Comment:
@clearlydefined/curation-dev is it really correct to repeat the `name`, `namespace`, `provider` and `revision` here? The [sourceLocation specs](https://github.com/clearlydefined/service/blob/b339cb7e281c1e35990b685efe9bfb774d9cd22f/schemas/curation-1.0.json#L135) state that all of `type`, `provider`, `name`, `revision` are required, but when using in the context of a curation it doe snot really make sense to repeat there, or?

Reply:
BTW, looks like this actually is the first example where the *curated* `url` points to a `sourcearchive` as no VCS repo is available.

Reply:
It makes sense. Please compare to a git example: https://github.com/clearlydefined/curated-data/blob/master/curations/maven/mavencentral/jakarta.ejb/jakarta.ejb-api.yaml